docs/terms/metric.md

```diff
@@ -4,12 +4,104 @@ title: Metric
 hoverText: A computed attribute of data such as the mean of a column.
 ---
 import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
 
-<UniversalMap setup='active' connect='active' create='active' validate='active'/>
+
+<UniversalMap setup='inactive' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
 
 A Metric is a computed attribute of data such as the mean of a column.
```

Comment:
I know we've talked about the notion that a Metric is just the answer to a question (what is the mean of my column, how many rows in my column are within the range 0<100), and I wonder whether we would want to mention that here at this stage. I believe Abe has worked on some content around this, and it felt like it could be major part of the narrative for GE.

Reply:
I've added that to the following section, since one of the Standards for theses docs is for the Definition section to mirror the Definition that shows up in hover text when a Technical Term is tagged in another document.